Dependencies/patches/ffmpeg-bcrypt-hacky-fix.patch

```diff
@@ -0,0 +1,11 @@
+--- a/libavutil/random_seed.c	2020-07-09 19:17:46.000000000 +1000
```

Comment:
It normally requires bcrypt.dll or .so and it would cause linking errors so I decided to forgo the entire bcrypt part, I do not believe that it will cause any errors as it was I think a noise dithering thing that needed it

Reply:
Hmm, cannot we just disable bcrypt in configure?